test/test_other.py

```diff
@@ -8480,7 +8488,41 @@ def test_run_order(self):
       foo();
     }
     ''')
-    self.do_runf('src.c', 'addAtInit\nctor\naddAtPostCtor\nmain\n', emcc_args=['--js-library', 'lib.js'])
+    # Each element is [<log message>, <applies to minimal runtime>]
+    expected_order = [
+      ['modulePreInit', False], # Not supported in minimal runtime.
+      ['modulePreRun', False], # Not supported in minimal runtime.
+      ['addOnPreRun', True],
+      ['addAtPreRun', True],
+      ['addOnInit', True],
+      ['addAtInit', True],
+      ['ctor', True],
+      ['addOnPostCtor', True],
+      ['addAtPostCtor', True],
+      ['addOnPreMain', True],
+      ['addAtPreMain', True],
+      ['main', True],
+      ['addOnExit', True],
+      ['addAtExit', True],
+      ['modulePostRun', False],  # Not supported in minimal runtime.
+      # The following two events are supported in miminal runtime, but they will
+      # not happen with MINIMAL_RUNTIME + EXIT_RUNTIME since it throws an
```

Comment:
Technically you don't need to use `EXIT_RUNTIME` here.. but then I guess you might not see other events like `addOnExit`?

Reply:
Yeah, I'm trying to test at much of the regular runtime ordering as I can.